static/js/hotkey.js

```diff
@@ -198,6 +198,8 @@ function process_hotkey(e) {
         } else if ($("#subscription_overlay").is(":visible")) {
             subs.close();
             return true;
+        } else if ($('#markdown-help').hasClass('in')) {
+            return false;
```

Comment:
@timabbott Can you please explain what is meant by 'search-help'? I am not able to get it clearly.

By the way, while searching for it myself, I discovered that the modal in 'Invite more users' is also having similar issue. It will also be taken care of in subsequent commits.